### DISCUSSION TITLE
How much CPU usage do you guys have?

### DISCUSSION THREAD

**Discussion by seeingrain** [jlesage/docker-freefilesync#15] [General]
![image](https://github.com/jlesage/docker-freefilesync/assets/7134524/b3c84f9d-2623-4c95-9a8e-639af7fe6822)

I have a NAS, brand is ZSpace, made in China. When I install freefilesync inside docker, the CPU usage is always above 3%. 
As a contrast, other docker container CPU usage stays 0~1% as most time.
Just wondering how you guys have? Is this usage normal?

**Comment by jlesage** [comment]
Mine is around 1.60%-1.75% (when looking at `docker stats`).

> As a contrast, other docker container CPU usage stays 0~1% as most time.

Are you comparing with other containers that implement a full desktop application ?

**Reply by seeingrain** [reply]
Hi, most containers I have are based on CLI, so cannot compare.
3% sounds not too much, forget it if not big deal. LOL.
